relay-dynamic-config/src/defaults.rs

```diff
@@ -98,6 +99,12 @@ fn span_metrics(is_extract_all: bool) -> impl IntoIterator<Item = MetricSpec> {
 
     let is_mobile = is_mobile_sdk.clone() & (is_mobile_op.clone() | is_screen);
 
+    let is_interaction = if is_extract_all {
+        RuleCondition::glob("span.op", "ui.interaction.*")
+    } else {
+        RuleCondition::never()
+    };
```

Comment:
@0Calories this means that span metrics for interaction spans will only exist for our own orgs for now. We have to remove this condition before GA'ing.

Reply:
Never mind, that feature flag is always enabled these days. https://github.com/getsentry/sentry-options-automator/pull/603